import_flow_events.py

```diff
@@ -61,8 +61,8 @@
 """
 Q_CREATE_METADATA_TABLE = """
     CREATE TABLE IF NOT EXISTS flow_metadata (
-      flow_id VARCHAR(64) NOT NULL UNIQUE ENCODE lzo,
```

Comment:
Do we lose anything by dropping the `UNIQUE` here?  Redshift doesn't enforce the uniqueness, but it does allegedly use it to speed up queries in some circumstances.  (Indeed, I wonder if `flow_id` should be marked as `PRIMARY KEY` in this table).